docs/sources/operators-guide/configuring/configuring-out-of-order-samples-ingestion.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Configuring out-of-order samples ingestion"
```

Comment:
One of the comment on other PR was what does "time window" exactly mean.
```
Same question as above, what does `old` mean in this context? does it mean:

a) older than wall clock time  
b) older than the newest ingested sample of this tenant  
c) older than the newest ingested sample of this ingester

i think it would be great if the help text would make it clear which of the above options it is
```

So we will have to explain how the time window is used. i.e. w.r.t. series if the series exist, or w.r.t. the tsdb's max time if the series does not exist. (technically it is head block's max time, but to keep it simple for the end user to understand we can use the term tsdb)

Reply:
I think I addressed this at the beginning of the file. Please review.